Libplanet/Blocks/Block.cs

```diff
@@ -261,6 +261,10 @@ public byte[] ToBencodex(bool hash, bool transactionData)
         /// any <see cref="IAction"/> of <see cref="Transactions"/> tries
         /// to update the states of <see cref="Address"/>es not included
         /// in <see cref="Transaction{T}.UpdatedAddresses"/>.</exception>
+        /// <exception cref="SimultaneousTxsException">Thrown when a
+        /// <see cref="Transaction{T}.Signer"/> has more than one
+        /// <see cref="Transaction{T}"/>s in <see cref="Transactions"/>.
```

Comment:
The last word seems misleading (it's not incorrect though).   How about rewording a simple term “block” here?

```suggestion
        /// <see cref="Transaction{T}"/>s in a block.
```

Reply:
Fixed.